src/libphonenumber/PhoneNumber.php

```diff
@@ -33,50 +56,101 @@ public function clearCountryCode()
     }
 
     /**
-     * @var int
+     * The national number.
+     *
+     * @var string|null
```

Comment:
Good point. It needed to set it as a String, use to floating point precision. Feel free to open another PR.